Extend/Model/Order.php

```diff
@@ -246,4 +248,109 @@ public function getTranslatedState()
     {
         return self::getTranslatedStates()[$this->oxorder__wdoxidee_orderstate->value] ?? '';
     }
+
+    /**
+     * {@inheritdoc }
+     * The emails will not be sent if module payment type is used
+     *
+     * @param object $oUser
+     * @param object $oBasket
+     * @param object $oPayment
+     *
+     * @return bool
+     */
+    protected function _sendOrderByEmail($oUser = null, $oBasket = null, $oPayment = null)
+    {
+        if ($this->isCustomPaymentMethod()) {
+            return self::ORDER_STATE_OK;
```

Comment:
The constant returned here is not boolean, but integer. Please verify, cast it to `bool` if necessary.

Reply:
OK, as I realised, no need for statuses here, as I want only indicate if emails were sent or not. Switched to booleans.